ambry-server/src/main/java/com/github/ambry/server/RepairRequestsSender.java

```diff
@@ -217,12 +216,30 @@ public void run() {
         }  // loop until all the partitions are clean
 
         // sleep for some time
-        Thread.sleep(SLEEP_TIME_MS);
+        if (!shutdown) {
+          Thread.sleep(SLEEP_TIME_MS);
+        }
       } catch (Throwable e) {
         metrics.repairSenderErrorHandleCount.inc();
         logger.error("RepairRequests Sender: Exception when handling request", e);
       }
     }
+
+    // shutdown the database
+    shutdownDb();
```

Comment:
The RepairRequestDB is passed to this class from AmbryServer, it's better to close it in AmbryServer.

Reply:
Closed the db in AmbryServer. Thanks